README.md

```diff
@@ -149,16 +129,18 @@ So, the following mock script:
 (when-mocking
    (f a) =1x=> a
    (f a) =2x=> (+ 1 a)
-   (g a b) => 32
+   (g a b) => 17
 
    (assertions
-     (+ (f 2) (f 2) (f 2) (g 1 2) (g 99 4) => 72))
+     (+ (f 2) (f 2) (f 2) 
+        (g 3e6 :foo/bar) (g "otherwise" :invalid) 
+     => 42))
 ```
 
 should pass. The first call to `f` returns the argument. The next two calls return the argument plus one.
-`g` can be called any amount (but at least once) and returns 32 each time.
+`g` can be called any amount (but at least once) and returns 17 each time.
 
-If you were to remove any call to `f` this test would fail.
+If you were to remove any call to `f` or `g` this test would fail.
 
```

Comment:
Added "Spies" section d00b2cc
Added "Protocols and Inline functions" section bc6c486